input.txt

```diff
@@ -1 +1 @@
-﻿
\ No newline at end of file
+﻿Test is in progress but it is not finished yet. Please be patient. Thank you.
```

Comment:
to test I would write few digits to check

Reply:
1 phase from slides, test basic use case, you could have tried it and saw that it is not fully working